### ISSUE TITLE
Dark runes shouldn't exist until 1.8

### ISSUE THREAD

**Issue by Scyllaqt** [LightsHope/issues#631]
Hey, Dark Runes are currently dropping on Northdale when they weren't added to vanilla until patch 1.8.

If you look at the [Allakhazam archive for the item](https://web.archive.org/web/20060306090437/http://wow.allakhazam.com:80/db/item.html?witem=20520), the earliest comments are from October 12th 2005, two days after the 1.8 patch launched.  This, to me at least, confirms the note at the bottom of the[ Wowpedia article](https://wow.gamepedia.com/Dark_Rune) on the item that claims it was added in 1.8.  Not sure if anyone has better evidence but I can't find anything that suggests they existed beforehand.

**Comment by Keftenk** [comment]
Correct. I think maybe LH loaded a incomplete version of the Patch progression from previous Ely/Nost builds.

Dark Rune 1.8 (Patch 6) documentation: https://web.archive.org/web/20060617173713/http://wow.allakhazam.com:80/db/item.html?witem=20520

**Comment by Gamemechanicwow** [comment]
A commit was done to the dev branch 5 days ago that should fix it. Please be patient until its pushed to live servers.

**Comment by ratkosrb** [comment]
Already fixed here https://github.com/lh-server/core/commit/ae2118346236eddb0cf10efc49fca004c31ca468
